c/tskit/tables.c

```diff
@@ -3553,6 +3589,9 @@ tsk_population_table_print_state(tsk_population_table_t *self, FILE *out)
         (int) self->metadata_length, (int) self->max_metadata_length,
         (int) self->max_metadata_length_increment);
     fprintf(out, TABLE_SEP);
+    fprintf(out, "#metadata_schema#\n");
+    fprintf(out, "%.*s\n", self->metadata_schema_length, self->metadata_schema);
+    fprintf(out, "#end#metadata_schema\n");
```

Comment:
Ouch, well spotted. I clearly need some tests to check the round tripping of schemas.

Reply:
Well this is just a print_state which is a debugging tool, so I wouldn't worry too much about it.